src/ILLink.Shared/MessageFormat.cs

```diff
@@ -20,5 +27,19 @@ public static string FormatRequiresAttributeUrlArg (string? url)
 				arg2 = " " + url;
 			return arg2;
 		}
+
+		public static string FormatRequiresAttributeMismatch (bool memberHasAttribute, bool condition, string var0, string var1, string var2)
```

Comment:
What does "condition" mean? Is there a better name?

Reply:
Changed for "isInterface"